cli/endpoints/online/triton/ensemble/create-endpoint-with-deployment-mir.yml

```diff
@@ -0,0 +1,26 @@
+name: mir-ensemble-model
+type: online
+auth_mode: aml_token
+traffic:
+    blue: 100
+deployments:
+    -   name: blue
+        model:
+           name: bidaf-ensemble
+           version: 1
+           local_path: ./models
+        code_configuration:
+            code:
+                local_path: ./code
+            scoring_script: dummy.py
```

Comment:
Can you just delete the code_configuration instead of providing code_configuration? I thought as part of BYOC we made this part of the YAML optional.

Reply:
Only for BYOC it is optional but the default deployment still requires code_configuration.

Reply:
That's annoying... can you create a backlog item to make it optional?

Reply:
There is already a backlog item.